simon/_posts/2017-10-12-day-31.md

```diff
@@ -13,4 +13,5 @@ Things to write about:
 * Did you do any special exercise??
 
 ***
-I went on a hike and did mental math.
+
+I went to the audobon center then I went on a hike and did mental math.
```

Comment:
"audobon" = Audubon (I keep making this mistake, too!)

Audubon Center should be capitalized.